src/views/Books/Books.js

```diff
@@ -0,0 +1,29 @@
+import React, { useState, useEffect } from 'react';
+import { useParams } from 'react-router-dom/cjs/react-router-dom.min';
+import { fetchBooks } from '../../services/books';
+
+export default function Books() {
+  const params = useParams();
+  const [books, setBooks] = useState([]);
+  const [loading, setLoading] = useState(true);
+  useEffect(() => {
+    const fetchData = async () => {
+      const data = await fetchBooks();
+      setBooks(data);
+      setLoading(false);
+    };
+    fetchData();
+  }, [params]);
```

Comment:
We actually do not need to call params here, only if we are pulling a parameter from the url, such as /:id